src/clj/rems/api/applications.clj

```diff
@@ -229,7 +229,8 @@
     (command-endpoint :application.command/change-resources commands/ChangeResourcesCommand)
     (command-endpoint :application.command/close commands/CloseCommand)
     (command-endpoint :application.command/decide commands/DecideCommand)
-    (command-endpoint :application.command/delete commands/DeleteCommand)
+    (command-endpoint :application.command/delete commands/DeleteCommand
+                      "Only drafts can be deleted. Only applicants can delete drafts.")
```

Comment:
Wow, didn't know we had this already :+1:

Reply:
yeah I was surprised too